frontend/packages/kubevirt-plugin/src/utils/sort.ts

```diff
@@ -46,3 +47,17 @@ export const ignoreCaseSort = <T>(
   };
   return array.sort((a, b) => resolve(a).localeCompare(resolve(b)));
 };
+
+export const removeOSDups = (osArr: OperatingSystemArray): OperatingSystemArray => {
+  const osNames: { [key: string]: string } = {};
+  osArr.forEach((os) => {
```

Comment:
what happens when these OS are in different templates?

Reply:
what do you mean?
we expect each record to be in the form: { name: "Fedora31", id: "someID" }.
Am I missing something ?

Reply:
if you have
template1: 
 { name: "Fedora31", id: "someID" }.

template2 
 { name: "Fedora32", id: "someID" }.

you will always choose template2 even though you should have chosen template1. Don't forget these templates can be also defined by the user.

Reply:
edited original comment 

ah I see, I made a mistake - now it makes sense even with mutiple templates - we just choose the newer one
and if the user wants his to be used he should change their own template to 

```
template1: 
 { name: "Fedora29", id: "fedora29" }.
```